range_queries/prefix_sum_array.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file prefix_sum_array.cpp
```

Comment:
Better to keep it like this: Doxygen will automatically check and set the filename. 🙂
```suggestion
 * @file
```